data/ecosystems/m/multiversx-elrond.toml

```diff
@@ -2001,3 +2001,198 @@ url = "https://github.com/zdenal/elrond-amm"
 
 [[repo]]
 url = "https://github.com/ziestwilky/elrond-smartcontract"
+
+[[repo]]
+url = "https://github.com/AIRO-NETWORK/airo-core-backend"
+
+[[repo]]
+url = "https://github.com/jimcoco/Blend-MultiversX"
+
+[[repo]]
+url = "https://github.com/combased-dev/calileo-frontend"
+
+[[repo]]
+url = "https://github.com/raresraulea/sw-licensing-api"
+
+[[repo]]
+url = "https://github.com/CheckerChain"
+
+[[repo]]
+url = "https://github.com/Chessout/chessout"
+
+[[repo]]
+url = "https://github.com/CoinDrip-finance/protocol-v2"
+
+[[repo]]
+url = "https://github.com/CosmicF?tab=repositories"
+
+[[repo]]
+url = "https://github.com/newtmex/CryptGain-submission-for-xDay-hack-2023"
+
+[[repo]]
+url = "https://github.com/radupara/decentralized-copyright-shield"
+
+[[repo]]
+url = "https://github.com/thedefibuilder/xdays-xdefigpt"
+
+[[repo]]
+url = "https://github.com/Entangle-Protocol/"
+
+[[repo]]
+url = "https://github.com/doinel1a/gen-x"
+
+[[repo]]
+url = "https://github.com/E-Compass/hackathon_portfolio_ui/"
+
+[[repo]]
+url = "https://github.com/GaupaLabs/Gaupa-Tracker-App"
+
+[[repo]]
+url = "https://github.com/GiveCup"
```

Comment:
This is an org link, not a repo link. Please add this org link to the org list section at the top of the file if you consider that the entire org is building / built in MultiversX.